test/val/pragma-once-sample-link.h

```diff
@@ -0,0 +1 @@
+pragma-once-sample.h
```

Comment:
adding symlinks to  a git repo breaks the repo for windows, as they dont exist there.

Reply:
Oh, my bad